transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_prod_performance_events.sql

```diff
@@ -0,0 +1,14 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['received_at_date'],
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+SELECT * FROM 
+{{ ref('stg_mm_telemetry_prod__performance_events') }}
+
+{% if is_incremental() %}
+    WHERE received_at_date >= (SELECT MAX(received_at_date) FROM {{ this }}) 
```

Comment:
Incremental strategy should rely on `received_at`.

Reply:
Updated to `received_at`